src/vma/event/event.h

```diff
@@ -35,6 +35,7 @@
 #define EVENT_H
 
 #include "vma/util/to_str.h"
+#include "utils/bullseye.h"
```

Comment:
nicer to keep include order as follows:
1. system includes on top (<typeinfo, stdio, stdint...)
2. general product includes second (utils/bullseye.h))
3. specific module includes last (vma/util/to_str.h)